app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -30,6 +30,16 @@
         clarinet: "<%= asset_path("instruments/clarinet.png") %>",
       }
     }
+
+    get lesson_card() {
+      return {
+        cancel: "<%= asset_path("lesson_card/cancel.png") %>",
+        location: "<%= asset_path("lesson_card/location.png") %>",
+        modify: "<%= asset_path("lesson_card/modify.png") %>",
+        person: "<%= asset_path("lesson_card/person.png") %>",
+        time: "<%= asset_path("lesson_card/time.png") %>",
```

Comment:
These icons are probably used across the website, could you put this under a icons() method to make it more generic?